mailpoet/assets/js/src/newsletters/send/sender-address-field.jsx

```diff
@@ -2,7 +2,7 @@ import { Component } from 'react';
 import PropTypes from 'prop-types';
 import { MailPoet } from 'mailpoet';
 import { FormFieldText } from 'form/fields/text.jsx';
-import { SenderEmailAddressWarning } from 'common/sender-email-address-warning.jsx';
+import { SenderEmailAddressWarning } from 'common/sender-email-address-warning.tsx';
```

Comment:
Not a big deal but I think we can simply drop the extension?

```suggestion
import { SenderEmailAddressWarning } from 'common/sender-email-address-warning';
```